src/scripts/components/VideoStatus.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import React, { Component } from 'react'
+// import styled from 'styled-components'
+// import Button from './foundations/Button'
+// import Input from './foundations/Input'
+import Wrapper from './foundations/Wrapper'
+
+type Props = {
+  video: Object,
+  videoFromPTI: Object
+};
+
+class VideoStatus extends Component<Props, void> {
+  render () {
+    let video = this.props.video
```

Comment:
I think we should prefer `const` over `let` whenever a variable is never re-assigned